docs/licensing.md

```diff
@@ -1,9 +1,364 @@
+
+**NOTE:** For deactivating licenses, please contact support.
+
+### Batch
+
+**Online Node-Locked Batch Activation**
+
+This activation mode *requires access to the internet* for activating licenses.
+
+Whenever activating AdonisFX for the first time for a specific DCC in batch mode, a product key has to be registered previously. Activating batch mode requires the use of an executable that eases the activation process of the product.
+
+To activate AdonisFX in Online Node-Locked Batch mode:
+
+1. Run `ActivateBatch` from `AdonisFX/bin` with admin privileges.
+2. Enter the `PRODUCT KEY`.
+    - The `PRODUCT KEY` has the following format: **XXXX-XXXX-XXXX-XXXX-XXXX-XXXX-XXXX**
+3. AdonisFX is activated and ready to be used.
+
+**Offline Node-Locked Batch Activation**
+
+This activation mode does *not require access to the internet* for activating licenses.
+
+To be able to activate AdonisFX in batch mode without relying on internet access an *Activation Request* and a *Activation Response* have to be generated.
+To ease this process several executable files are provided to generate the associated XML files for activation.
+
+To activate AdonisFX in Offline Node-Locked Batch mode:
+
+1. Open `AdonisFX/bin`.
+2. Run `OfflineRequest` from `AdonisFX/bin` with admin privileges.
+3. Select **Batch** mode by entering the value `1`.
+4. Enter a valid product key.
+    - A XML file `BatchOfflineRequest.xml` will be generated in the same folder.
+
+5. Send the request file to **adnsupport@inbibo.co.uk** providing enough information to backtrack the source of the activation request (username and date).
+6. In a maximum of 24h a `BatchOfflineResponse.xml` will be returned to the source e-mail address.
+7. Download and save the response in the same folder `AdonisFX/bin`.
+8. Execute `OfflineReponse` from `AdonisFX/bin` with admin privileges.
+9. Select **Batch** mode again by entering the value `1`.
+10. The response will be registered and AdonisFX will be ready to be used.
+    - The `PRODUCT KEY` has the following format: **XXXX-XXXX-XXXX-XXXX-XXXX-XXXX-XXXX**
+    - Once the activation period has concluded, a new activation request and response have to be generated.
+
+**NOTE:** For deactivating licenses, please contact support.
+
+### Trial
+
+AdonisFX allows the user to use the product for **30 days** in Node-Locked Interactive mode. This means that the trial can be used using the graphical interface for one single machine at a time. The trial period requires activation which can be handled in an online or offline way.
+
+Trial licenses are intended for testing and non-commercial purposes. To use AdonisFX for commercial purposes a `PRODUCT KEY` must be purchased through Inbibo's official website [https://www.inbibo.co.uk/](https://www.inbibo.co.uk/) and activated.
+
+To request an extension of the trial license period contact support at **adnsupport@inbibo.co.uk**.
+
+See the **AdonisFX EULA** for details.
```

Comment:
Should we link here the EULA or remove this line?

Reply:
There will be a link in the website too. Add a "PENDING" for now

Reply:
Done!